libs/dh/metering-point/feature-overview/src/lib/dh-metering-point-overview.component.spec.ts

```diff
@@ -0,0 +1,67 @@
+/**
+ * @license
+ * Copyright 2021 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Component } from '@angular/core';
+import {
+  dhMeteringPointIdParam,
+  DhMeteringPointOverviewGuard,
+  dhMeteringPointPath,
+} from '@energinet-datahub/dh/metering-point/routing';
+import { render, RenderResult, screen } from '@testing-library/angular';
+import { SpectacularAppComponent } from '@ngworker/spectacular';
+
+import { DhMeteringPointOverviewModule } from './dh-metering-point-overview.module';
+import { DhMeteringPointOverviewComponent } from './dh-metering-point-overview.component';
+
+describe(DhMeteringPointOverviewComponent.name, () => {
+  beforeEach(async () => {
+    view = await render(SpectacularAppComponent, {
+      imports: [DhMeteringPointOverviewModule],
+      routes: [
+        {
+          path: dhMeteringPointPath,
+          children: [
+            {
+              canActivate: [DhMeteringPointOverviewGuard],
+              component: DhMeteringPointOverviewComponent,
+              path: `:${dhMeteringPointIdParam}`,
+            },
```

Comment:
suggestion: consider how we could test the real route here so that the actual route is covered by a test and we don't have to keep actual routes and test routes in sync

Reply:
Could a solution be to move all routing setup from `DhMeteringPointShellModule` to `metering-point/routing` library? Then expose a `*ChildRoutesModule` that we import back to shell library. Then the `*ChildRoutesModule` can also be imported in other places where routing is necessary. Or am I misunderstanding the idea?

Reply:
When I was looking at the current solution last week, I didn't see a need for a routing library at all. I think we could have both the guard and the route path in its corresponding feature library but expose the route path. This would allow us to have integration tests in this library alone covering actual routes, route guards, modules, and components.